pilot/pkg/serviceregistry/kube/controller/network.go

```diff
@@ -63,6 +63,11 @@ func (c *Controller) reloadNetworkLookup() {
 		}
 	}
 	c.ranger = ranger
+	// the network for endpoints are computed when we process the events; this will fix the cache
+	// TODO(landow) there may be a race between this and the full push we trigger on the networks watcher.
+	if err := c.SyncAll(); err != nil {
```

Comment:
Do we need to sync all? Maybe just eds is ok?

Reply:
https://github.com/istio/istio/pull/26249